doc/Language/variables.pod6

```diff
@@ -659,7 +658,7 @@ perl6 -e '{ say ++$; say $++  } for ^5'
 =end code
 
 If you need to refer to the value of C<$> more than once within the
-scope it should be copied to a new variable.
+scope, it should be copied to a new variable.
```

Comment:
How about:
To refer to the value of C<$> more than once within a scope, copy the value to a new variable.